Tests/StreamChatTests/StateLayer/ChannelList_Tests.swift

```diff
@@ -230,15 +230,17 @@ final class ChannelList_Tests: XCTestCase {
     // MARK: - Test Data
     
     /// For tests which rely on the channel updater to update the local database.
-    private func setUpChannelList(usesMockedChannelUpdater: Bool, dynamicFilter: ((ChatChannel) -> Bool)? = nil) {
+    @MainActor private func setUpChannelList(usesMockedChannelUpdater: Bool, loadState: Bool = true, dynamicFilter: ((ChatChannel) -> Bool)? = nil) {
         channelList = ChannelList(
-            initialChannels: [],
             query: ChannelListQuery(filter: .in(.members, values: [memberId]), sort: [.init(key: .createdAt, isAscending: true)]),
             dynamicFilter: dynamicFilter,
             channelListUpdater: usesMockedChannelUpdater ? env.channelListUpdaterMock : env.channelListUpdater,
             client: env.client,
             environment: env.channelListEnvironment
         )
+        if loadState {
+            _ = channelList.state
+        }
```

Comment:
I would prefer not to force load it since it is going to look a bit ugly in inits and sometimes it might be beneficial as well. Create an instance, call some method, throw away the instance (loading state might be skipped at all).
But it is important in tests since there we have predefined flows. Test coverage should improve as well over time.